helm/e2e-app-chart/templates/deployment.yaml

```diff
@@ -9,7 +9,8 @@ spec:
   replicas: 2
   revisionHistoryLimit: 3
   strategy:
-    type: Recreate
+    type: RollingUpdate
+    maxUnavailable: 1
```

Comment:
It is not really adding nothing now. But if we create a complex test in the future where app deployment is updated during the upgrade process it can help.